index.html

```diff
@@ -1,35 +1,53 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" type="text/css" href="main.css">
-    <title>Web Development</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="main.css">
+  <title>Web Development</title>
 </head>
+
 <body>
-  <header><!--logo goes here--></header>
-    <div class="container">
-        <h1>Why Web Development?</h1> <!--sample title heading-->
-            <h3>Best career opportunity</h3>
-              <p>Everyone either needs a website or new features for it. Day by day increasing number of internet users and the continuously increasing number of websites have resulted in the high requirement for qualified web development professionals in India as well as abroad. Their skills and services are in demand in all sectors.</p>
-            <h3>Work from where you want</h3>
-              <p>Freedom is the most important reason to learn web development course. The internet is everywhere. The laptop is a portable device and you can use it anywhere. Websites can be reached from all over the world. Expert web developers benefit from a great amount of freedom when, where and how they work.</p>
-            <h3>Bright future scope</h3>
-              <p>Web development is the fastest-growing sector of Software development. It has the best career path with rapid growth and a handsome salary package, providing high job security and luxury of life. The Bureau of Labor Statistics said that approx 27 % of employment growth will be for web developers by 2020. During this time, approx 39,500 new jobs are for web developers in an industry that already has roughly 148,500 positions. According to CWJobs, 57% of London based web developers getting more than £50,000 a year and 40 % of contracted positions boasting over £100,000, web development is an attractive career path</p>
-      <div>
-          <!--Images Can Go Here-->
-        </div>
-    </div>
-    <div id="footer">
-      <h3 class="foothead">Web Development Team :</h3>
-      <ul>
-        <li><a href="https://github.com/JLevelston">JaNiece</a></li>
-        <li><a href="https://github.com/thetechtakeover">Evan</a></li>
-        <li><a href="https://github.com/codingpixi">Detra</a></li>
-        <li><a href="https://github.com/dontbecypher16">Richard</a></li>
-        <li><a href="https://github.com/nojocode">Noah</a></li>
-      </ul>
+  <header>
+
+    <img class="logo front logoDisplay" src="img/logo.png" alt="Black Orlando Tech logo with a robot head in place of the letter O">
+
+  </header>
+
+  <div class="container">
+    <h1>Why Web Development?</h1>
+    <!--sample title heading-->
```

Comment:
🙃 Not sure if this comment here is necessary?

Reply:
Ok. I will remove that now. Thx 😊